spec/models/project_duplicator_spec.rb

```diff
@@ -35,14 +35,30 @@
       create(:loan, :with_loan_media, :with_timeline, :with_accounting_transaction,
         :with_copies)
     end
-    let(:new_loan) { duplicator.duplicate }
+
+    # root_timeline_entry children are incorrect on new_loan. Reload it, to bust the cache.
+    # Using #reload does not do it.
+    let(:new_loan) { Loan.find(duplicator.duplicate.id) }
 
     it 'ignores media' do
       expect(new_loan.media.count).to eq 0
     end
 
-    it 'ignores timeline_entries' do
-      expect(new_loan.timeline_entries.count).to eq 0
+    it 'copies timeline_entries' do
```

Comment:
On the second pass, I will add an expectation to ensure the fields are being copied. No point in adding it before hand.

Reply:
Ok. When you are doing so, please be sure to add a test that the translations are being copied properly. I don't think we covered that this time and it's a big question mark for me at this point.